cgp/ea/mu_plus_lambda.py

```diff
@@ -159,20 +160,29 @@ def _sort(self, combined: List[Individual]) -> List[Individual]:
             if np.isnan(ind.fitness):
                 ind.fitness = -np.inf
 
-        # get list of indices that sorts combined_copy ("argsort")
+        # get list of indices that sorts combined_copy ("argsort") in descending order
```

Comment:
i would move this comment below. mentally i tend to skip functions until i see where there are used and that's where i would need the information that line 173 does argsort

Reply:
Done